packages/drivers/local-driver/src/localDocumentServiceFactory.ts

```diff
@@ -26,6 +26,63 @@ import { defaultHash } from "@fluidframework/server-services-client";
 import { LocalDocumentDeltaConnection } from "./localDocumentDeltaConnection";
 import { createLocalDocumentService } from "./localDocumentService";
 
+export class CreateDocumentInStorage {
+	private static instance: CreateDocumentInStorage | undefined;
+
+	private constructor(
+		private readonly localDeltaConnectionServer?: ILocalDeltaConnectionServer,
```

Comment:
Would it be possible to avoid this class by including this info here?
https://github.com/microsoft/FluidFramework/blob/cb7defaf0a88a0c325ae2d33fdd6d13c7b283edc/packages/drivers/local-driver/src/localDocumentService.ts#L51-L60

Reply:
I'll see if I can get all parameter in CreateDocumentInStorage class constructor in the LocalDocumentStorageService constructor.